### ISSUE TITLE
The loss function does not decrease

### ISSUE THREAD

**Issue by foolishman-wx** [ChuhaoZhou99/SAADG_VVIReID#2]
Dear Author: Hello, when I run your code, I find that the loss function does not descend and the training process does not converge. My environment is toch1.11,torch-geometric 2.1. What is the reason? Or can you give the trained model parameters? thank you.

**Comment by ChuhaoZhou99** [comment]
Hi there, sorry for the late responses and thank you for interesting in our work. 

We have re-checked the code and the training process goes well on a single GPU. Please note that the released code is temporarily not support for multi-GPUs training. It might be a possible explanation to your problem. We will extend the code to support multi-GPUs training in the future.

Besides, we have released the trained model weights, please refer to the [link](https://drive.google.com/file/d/1oQ-zUZfAKTctBrmSOXVP_QI8_V4Uf0Q_/view?usp=drive_link) to download.
